src/features/dsComponents/body/BodyTable.tsx

```diff
@@ -1,226 +1,219 @@
-import React from 'react'
+import React, { useState, useRef } from 'react'
+import { InfiniteLoader, Grid, MultiGrid, IndexRange } from 'react-virtualized'
+import ContentLoader from 'react-content-loader'
 import classNames from 'classnames'
-import ContentLoader from "react-content-loader"
+import useDimensions from 'react-use-dimensions'
+import 'react-virtualized/styles.css'
 
+import { QriRef } from '../../../qri/ref'
 import DataType from '../../../chrome/DataType'
 
+const PAGE_SIZE = 100
```

Comment:
We should rename the `bodyPageSizeDefault` const in `/src/features/dataset/state/datasetActions.ts` to `DEFAULT_BODY_PAGE_SIZE`

Then then we should replace `PAGE_SIZE` with importing `DEFAULT_BODY_PAGE_SIZE`